internal/examples/proto/buf.gen.yaml

```diff
@@ -9,6 +9,11 @@ managed:
 
 plugins:
   - name: go
-    out: gen
+    out: gen/go
     opt: module=go.einride.tech/protobuf-bigquery/internal/examples/proto/gen
     path: ./build/protoc-gen-go
+
+  - name: json_schemas
+    out: gen/json
+    opt: indent=true
+    path: ./build/protoc-gen-bq-json-schema
```

Comment:
update editor config to add trailing newline

Reply:
Ah! Well spotted. Got a new laptop this week and I'm missing my old Goland config. Will set that up and fix. 👍🏼